pyproject.toml

```diff
@@ -0,0 +1,88 @@
+[project]
+name = "locopy"
+description = "Loading/Unloading to Amazon Redshift using Python"
+readme = "README.rst"
+authors = [
+  { name="Faisal Dosani", email="faisal.dosani@capitalone.com" },
+]
+license = {text = "Apache Software License"}
+dependencies = [
+    "boto3<=1.28.3,>=1.9.92",
+    "PyYAML<=6.0,>=5.1",
+    "pandas<=2.0.3,>=0.25.2",
+    "numpy<=1.25.1,>=1.22.0",
+]
+
+requires-python = ">=3.8.0"
+classifiers = [
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
One minor nitpick, we should maybe add in 3.11 to the testing matrix and other checks?

Reply:
Oh yeah, good call! Missed that. Adding that now.